### ISSUE TITLE
Installation fails due to missing requirements folder

### ISSUE THREAD

**Issue by waza-ari** [overhangio/mu#1]
Very interesting project, I like it!

The installation currently fails as `setup.py` dynamically imports the requirements, but the requirements folder is missing from the distribution:

```
$ mu-courses-0.1.0 tree .
.
├── LICENSE.txt
├── PKG-INFO
├── README.md
├── mu
│   ├── __init__.py
│   ├── exceptions.py
│   ├── formats
│   │   ├── __init__.py
│   │   ├── base
│   │   │   ├── __init__.py
│   │   │   ├── reader.py
│   │   │   └── writer.py
│   │   ├── html
│   │   │   ├── __init__.py
│   │   │   ├── common.py
│   │   │   ├── reader.py
│   │   │   └── writer.py
│   │   ├── md
│   │   │   ├── __init__.py
│   │   │   ├── reader.py
│   │   │   └── writer.py
│   │   └── olx
│   │       ├── __init__.py
│   │       ├── reader.py
│   │       └── writer.py
│   ├── main.py
│   ├── units.py
│   └── utils
│       ├── __init__.py
│       └── youtube.py
├── mu_courses.egg-info
│   ├── PKG-INFO
│   ├── SOURCES.txt
│   ├── dependency_links.txt
│   ├── entry_points.txt
│   ├── requires.txt
│   └── top_level.txt
├── setup.cfg
└── setup.py
```

**Comment by macpijan** [comment]
I'm very interested in the project as well.

Do you mean such installation failure:

```
pip install mu-courses

Collecting mu-courses
  Using cached mu-courses-0.1.0.tar.gz (29 kB)
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error
  
  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [8 lines of output]
      Traceback (most recent call last):
        File "<string>", line 2, in <module>
        File "<pip-setuptools-caller>", line 34, in <module>
        File "/tmp/pip-install-32wa1lk8/mu-courses_522894d02ec141eda18de047c98b1a97/setup.py", line 43, in <module>
          install_requires=load_requirements("base.in"),
        File "/tmp/pip-install-32wa1lk8/mu-courses_522894d02ec141eda18de047c98b1a97/setup.py", line 16, in load_requirements
          with io.open(
      FileNotFoundError: [Errno 2] No such file or directory: '/tmp/pip-install-32wa1lk8/mu-courses_522894d02ec141eda18de047c98b1a97/requirements/base.in'
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
```

Were you able to go past through that?

**Comment by waza-ari** [comment]
Yeah, that's the error. Except for cloning and installing from there, there's no way to install the package as published AFAIK.

The problem is in the `setup.py` script: https://github.com/overhangio/mu/blob/5ad4a809f9e7d7d1868ed0cddea339bba2267ceb/setup.py#L17

It tries to load the base requirements, but they're not part of the distribution, so installation fails. We'd need to tell python to include the requirements folder in the distribution when using this command (copied from Readme): `python setup sdist`.

This however affects the published package, so only @regisb can actually fix this. I'd propose a MR, but I'm not too familiar with the "legacy" setup.py / dist stuff, mostly using Poetry these days to publish my packages. I fear I'd not be a lot of help here.

**Comment by regisb** [comment]
Thanks for your feedback @waza-ari @macpijan! This issue should now be resolved in v0.1.1.
